textproc/asciidoc/Portfile

```
@@ -4,11 +4,11 @@ PortSystem          1.0
 
 PortGroup           github 1.0
 
-github.setup        asciidoc asciidoc-py3 9.0.0rc2
+github.setup        asciidoc asciidoc-py3 9.0.0
```

Unfortunately since MacPorts considers `9.0.0rc2` > `9.0.0` you must add:

```suggestion
github.setup        asciidoc asciidoc-py3 9.0.0
epoch               1
```

---

Ouch, thanks for catching that. It turns out 9.0.1 was released a few hours ago, so might as well try that instead.